test/sql/test_array_fn/T/test_array_fn

```diff
@@ -825,3 +825,19 @@ select array_intersect([row(1,2,3), row(3,4,5)], [row(3,4,5)]);
 select array_contains_all([row(1,2,3), row(3,4,5)], [row(3,4,5)]);
 select array_filter([row(1,2,3), row(3,4,5), row(4,5,6)], [0,1,0]);
 select cardinality([row(1,2,3), row(3,4,5)]);
+
+select array_contains_seq([1,2,3,4], [2,3]);
```

Comment:
what if SELECT array_contains_seq([1, 2, NULL, 3, 4], ['a'])

Reply:
SELECT array_contains_seq([1, 2, NULL, 3, 4], [2,3]);
SELECT array_contains_seq([1, 2, NULL, 3, 4], null);
SELECT array_contains_seq(null, [2,3])
SELECT array_contains_seq([1, 2, NULL, 3, 4], [null,null])
SELECT array_contains_seq([1, 2, NULL], [null,2])
SELECT array_contains_seq(null, null)

Reply:
we prefer keep the same behavior with trino

Reply:
> what if SELECT array_contains_seq([1, 2, NULL, 3, 4], ['a'])

MySQL [(none)]> select array_contains_all([1, 2, NULL, 3, 4], ['a']);
+-----------------------------------------------+
| array_contains_all([1, 2, NULL, 3, 4], ['a']) |
+-----------------------------------------------+
|                                             0 |
+-----------------------------------------------+
1 row in set (0.18 sec)

Reply:
> SELECT array_contains_seq([1, 2, NULL, 3, 4], [2,3]); SELECT array_contains_seq([1, 2, NULL, 3, 4], null); SELECT array_contains_seq(null, [2,3]) SELECT array_contains_seq([1, 2, NULL, 3, 4], [null,null]) SELECT array_contains_seq([1, 2, NULL], [null,2]) SELECT array_contains_seq(null, null)

all add to sqltest

Reply:
here are all cases that just consider single array, please add some cases for array columns from a table, which would get some bugs.  like following cases

```
CREATE TABLE array_test ( 
pk bigint not null ,
i_0   Array<BigInt>,
i_1   Array<BigInt>,
ai_0  Array<Array<BigInt>>,
ai_1  Array<Array<BigInt>>
) ENGINE=OLAP
DUPLICATE KEY(`pk`)
DISTRIBUTED BY HASH(`pk`) BUCKETS 3
PROPERTIES (
"replication_num" = "1",
"in_memory" = "false"
);

insert into array_test values
(1,[null,1],[null],[null],[[]]),
(2,[null],[1,3,4,5,1,2],[[null,1],null],[[1,null]]),
(3,[],[],[[],null,[1,1]],[],[1,1]),
(4,null,[],[[1,1]],[[1,1],null),
(5,[4,4,4],[4,null],[null],[null]),
(6,[1,1,2,1,1,2,3,3],[1,2,3],[[1]],[[1],[2],null,[null]]);

select array_contains_seq(i_0,i_1),array_contains_seq(i_0,i_0),array_contains_seq(i_1,i_0), array_contains_seq(ai_0, ai_1),array_contains_seq(ai_1, ai_1),array_contains_seq(ai_1, ai_0),array_contains_seq(ai_0, i_1) from array_test;

```

Reply:
add this case to the Test file